### ISSUE TITLE
Screen recording delay

### ISSUE THREAD

**Issue by 4ctivated** [ShareX/ShareX#4833]
Hello!

I very very very love this software and I have a big trouble with delay at screen recording.

There are few examples:

- always move mouse 0-10 sec https://my.adminvps.ru/bz/man/install/oJlCK4vGA1.gif

as u can see, delay at start about 3-4 sec.

- writing 0-10 sec https://my.adminvps.ru/bz/man/install/16CUuOnLJE.gif

as u can see, there is a cut of few seconds at start.

Could you please check debug log and help me?
[debuglog.txt](https://github.com/ShareX/ShareX/files/4695917/debuglog.txt)

I think the problem with ffmpeg may be.

All worked well before today.

**Comment by Jaex** [comment]
All the time peoples FFmpeg related issues happens like this: "All worked well before today."
But ShareX is like last updated 3 months ago. So how randomly your FFmpeg recording becomes like this I don't understand. Something in your PCs must have changed which causing this but I can't figure out what.

Did you tried updating FFmpeg using ShareX?

**Comment by 4ctivated** [comment]
Thank you VERY MUCH Jaex for fast reply <3

I'm Dmitry from Belarus.

> Did you tried updating FFmpeg using ShareX?

No, i don't. I need to do it?

I would even be happy to roll back the system a week ago, if only this wonderful program worked, but I did not create any recovery points :(
I'm not sure if this is ffmpeg, but that’s quite possible, right?
Now I will try to update ffmpeg.

**Comment by Jaex** [comment]
Some users issue resolved after update FFmpeg using ShareX.

**Comment by 4ctivated** [comment]
> Some users issue resolved after update FFmpeg using ShareX.

Hello!
I can't find update feature in software. But i've find out the reason of delay.
It was antivirus software. I've stopped antivirus and all looks good now! Thank you :)

**Comment by Jaex** [comment]
Which anti virus software are you using? Because a lot of people reporting this issue and we are unable to figure out the cause of it.

**Comment by Dreamykass** [comment]
I also experience this issue, and I found that it may or may not be because of Avast Antivirus. See the other issue I mentioned this issue from.

I have the program in polish and not in english, so names of options might not be the most accurate. Sorry. Things I tried:
- Nothing touched - freezing is present.
- Right-click on avast icon, shields control, disable for 10 minutes - **freezing is not present**. Hooray?
- Folder that sharex is installed in added to exceptions for scanning (?) - no effect.
- Folder that sharex saves videos and images onto added to exceptions for scanning (?) - no effect.
- Shields, first and second one disabled, https://i.imgur.com/bOSZmty.png - no effect.
- Scanning of programs and files disabled, https://i.imgur.com/Q2dliNl.png, - no effect.

Sorry for the ping, but as you said "a lot of people reporting this issue and we are unable to figure out the cause of it", that looks to me like it'd be nice to help as quickly as possible. @Jaex 

I'd be willing to help with troubleshooting and finding the cause of and/or solution to the problem, yeah.
